time_tracker_api/technologies/technologies_api.py

```diff
@@ -0,0 +1,83 @@
+from flask_restplus import Namespace, Resource, fields
+
+ns = Namespace('technologies', description='API for technologies used')
+
+# Technology Model
+technology = ns.model('Technology', {
+    'id': fields.String(
+        readOnly=True,
+        required=True,
+        title='Identifier',
+        description='The unique id of the technology'
+    ),
+    'tenant_id': fields.String(
+        required=True,
+        title='Tenant',
+        max_length=64,
+        description='The tenant this technology belongs to'
+    ),
+    'name': fields.String(
+        required=True,
+        title='Name',
+        max_length=50,
+        description='Name of the technology'
+    ),
+    'created_at': fields.Date(
+        title='Date',
```

Comment:
I would like that these kinds of fields are spread equally in the rest of the models so `Date` is a little ambiguous, it could be confused with the start date of the time entries for example. I would leave as the title "Created" and the description is great.